docs/docusaurus.config.js

```diff
@@ -11,7 +11,7 @@ const config = {
     title: 'BigBlueButton',
     tagline: 'Official Documentation',
     url: 'https://docs.bigbluebutton.org/',
-    baseUrl: isDev ? '/docs' : '/',
+    baseUrl: '/',
```

Comment:
The `/docs` exists for some reason.
Have you trying to run the docs using `./run-dev.sh`?

![image](https://github.com/user-attachments/assets/7bb403d5-f88f-48d0-a2c3-37ecfb224872)

It suggests a link that requires the `/doc` config to work.

Reply:
For some reason I had to tweak `docusaurus.config.js` with this config to manage `run-dev.sh` to work:

```
docs: {
                    routeBasePath: "/",
                    sidebarPath: require.resolve('./sidebars.js'),
                    lastVersion: 'current',
                    includeCurrentVersion: true,
                    versions: {
                        'current': {
                            banner: 'none'
                        },
                    }
                },
```

It was not necessary in the past, I don't know what broke versions in dev env.

Reply:
> It was not necessary in the past, I don't know what broke versions in dev env.
Hm, to my knowledge, the versions are not built by default. Did you run `./build.sh`? This builds all the versions normally (It was working locally for me - not on docker environment, at least).